docs/release-notes/release-notes-1.6.0-rc1.1.md

```diff
@@ -0,0 +1,925 @@
+transaction.
+
+### HTTPS Seeding
+
+The initial bootstrap process that contacts seeders to discover other nodes to
+connect to now uses a REST-based API over HTTPS.
+
+This change will be imperceptible for most users, with the exception that it
+accelerates the process of finding suitable candidate nodes that support desired
+services, particularly in the case of recently-introduced services that have not
+yet achieved widespread adoption on the network.
+
+The following are some key benefits of HTTPS seeders over the previous DNS-based
+seeders:
+
+- Support for non-standard ports
+- Advertisement of supported service
+- Better scalability both in terms of network load and new features
+- Native support for TLS-secured communication channels
+- Native support for proxies which allows the use of anonymous overlay networks
+  such as Tor and I2P
+- No need for a large DNSSEC dependency surface
+- Uses better audited infrastructure
+- More secure
+- Increases flexibility
+
+### Signature Validation And Other Crypto Operation Optimizations
+
+The underlying crypto code has been reworked to significantly improve its
+execution speed and reduce the number of memory allocations.  While this has
+more benefits than enumerated here, probably the most important ones for most
+stakeholders are:
+
+- Improved vote times since blocks and transactions propagate more quickly
+  throughout the network
+- The initial sync process is around 15% faster
+
+### Proactive Signature Cache Eviction
+
+Signature cache entries that are nearly guaranteed to no longer be useful are
+now immediately and proactively evicted resulting in overall faster validation
+during steady state operation due to fewer cache misses.
+
+The primary purpose of the cache is to avoid double checking signatures that are
+already known to be valid.
+
+### Orphan Transaction Relay Policy Refinement
+
+Transactions that spend outputs which are not known to nodes relaying them,
+known as orphan transactions, now have the same size restrictions applied to
+them as standard non-orphan transactions.
+
+This ensures that transactions chains are not artificially hindered from
+relaying regardless of the order they are received.
+
+In order to keep memory usage of the now potentially larger orphan transactions
+under control, more intelligent orphan eviction has been implemented and the
+maximum number of allowed orphans before random eviction occurs has been
+lowered.
+
+These changes, in conjunction with other related changes, mean that nodes are
+better about orphan transaction management and thus missing ancestors will
+typically either be broadcast or mined fairly quickly resulting in fewer overall
+orphans and smaller actual run-time orphan pools.
+
+### Ticket Exhaustion Prevention
+
+Mining templates that would lead to the chain becoming unrecoverable due to
+inevitable ticket exhaustion will no longer be generated.
```

Comment:
Might want to mention what "ticket exhaustion" is in parens, i.e. is it a condition when ticket pool drops below some X?

Reply:
The general goal is to keep this pretty high level (despite my inclinations to the contrary) and explaining the details there would necessarily get pretty technical.